includes/Rtree.h

```diff
@@ -0,0 +1,26 @@
+#ifndef RTREE_H
+#define RTREE_H
+
+#include "Node.h"
+
+// create a R-tree with a root node
+class Rtree {
+public:
+    Rtree();
+    ~Rtree();
+
+    Rtree(Node*);
+
+    const Node* getRoot() const { return root; }
+    const int getSize() const { return size; }
+
+    void insert(Rect);
+    void remove(Rect);
+    void search(Rect);
+
+private:
+    Node* root = nullptr;
+    int size = 0;
```

Comment:
Better provide some comments about what does this mean.

The variable name `size` is bad because conventionally we use `size()` to name a member function for any class for which sizing makes sense.  This is yet another example why we always prefix `m_` to member data.

Reply:
Done!